src/launchAsync.ts

```diff
@@ -126,3 +126,18 @@ export function launchAsync(token: string, content: Content, options?: Options):
         document.documentElement.style.overflow = 'hidden';
     });
 }
+
+/**
+ * Style the launch buttons with an encoded SVG of the assets\icon.svg
+ */
+export function styleLaunchButtons(): void {
```

Comment:
Add enum for the type of button (e.g. image only, image with text, text only). For the "Immersive Reader" string, might make sense to hardcode all the localized strings in the TS somewhere.

Reply:
Will add that extended functionality later